cli/pkg/github/git.go

```diff
@@ -0,0 +1,47 @@
+package github
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing/transport/ssh"
+)
+
+func PublicKey() (*ssh.PublicKeys, error) {
+	var publicKey *ssh.PublicKeys
+	sshPath := os.Getenv("HOME") + "/.ssh/id_ed25519"
+	sshKey, _ := os.ReadFile(sshPath)
+	publicKey, err := ssh.NewPublicKeys("git", sshKey, "")
+	if err != nil {
+		return nil, err
+	}
+	return publicKey, err
+}
+
+func CloneRepo(url string) (string, error) {
+	repoName := strings.TrimSuffix(url[strings.LastIndex(url, "/")+1:], ".git")
+	fmt.Printf("Cloning Repo URL: {%s}\n", url)
```

Comment:
Usually, `git` will output `Cloning into 'repo-name'...`. If the `Progress: os.Stdout` doesn't do that, our print statement here should match that format.

Reply:
its not logging such msg, so adding same in out print statement 👍